### PR TITLE
Update uninstall.md

### PR DESCRIPTION
**Pull request by timothywarner**
Provided a link to the wusa tool.

### CODE REVIEW

File: wmf/uninstall.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Uninstallation Instructions
+# Uninstalling WMF v5
```
Comment (HemantMahawar) [inline wmf/uninstall.md]: This change is not necessary as the uninstallation instructions are in context of WMF 5 only.

File: wmf/uninstall.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 ## Using Command Prompt
 1.	Open **Command Prompt.**
-2.	Run the following command:
+2.	Run the [Windows Update Standalone Launcher](https://support.microsoft.com/en-us/kb/934307)as shown below:
```
Comment (HemantMahawar) [inline wmf/uninstall.md]: Missing space between link and the word "as"

**Comment by HemantMahawar** [comment]
@timothywarner can you address these 2 comments?

**Comment by timothywarner** [comment]
Hi Hemant. Sure, I'll address these two comments tonight when I get home from work (I'm in Central Time). Thanks, Tim

**Comment by timothywarner** [comment]
Ok, Helmant. I edited the file in my branch directly on GitHub. I'm still a newb with Git and GitHub--let me know if my edits haven't carried through to your side. Thanks, Tim

**Comment by joeyaiello** [comment]
Looks good to me! @HemantMahawar if you're good with the changes, feel free to merge.

**Comment by timothywarner** [comment]
Hey Joey and Hermant, it would be really helpful to me if you could call out specific documents in the repo that require the most attention. At the moment I'm simply aimlessly spelunking. Happy New Year! -Tim

**Comment by joeyaiello** [comment]
@timothywarner I would definitely start with the [Issues](https://github.com/PowerShell/PowerShell-Docs/issues). However, you raise a valid point that we could use a higher-level backlog (or at least some higher-level items in our Issues). Duly noted.

**Pull request merged**